ding/reward_model/trex_reward_model.py

```diff
@@ -99,10 +100,10 @@ def __init__(self, config: EasyDict, device: str, tb_logger: 'SummaryWriter') ->
         self.learning_rewards = []
         self.training_obs = []
         self.training_labels = []
-        self.num_trajs = 0 # number of downsampled full trajectories
-        self.num_snippets = 6000 # number of short subtrajectories to sample
-        self.min_snippet_length = 50 # minimum number of short subtrajectories to sample
-        self.max_snippet_length = 100 # maximum number of short subtrajectories to sample
+        self.num_trajs = 0  # number of downsampled full trajectories
+        self.num_snippets = 6000  # number of short subtrajectories to sample
+        self.min_snippet_length = 50  # minimum number of short subtrajectories to sample
+        self.max_snippet_length = 100  # maximum number of short subtrajectories to sample
```

Comment:
It seems better to put these parameters to config because the length may be affected by CUDA's memory.

Reply:
Thx! that is a great suggestion, I will put these into the config.reward_model